orttraining/orttraining/python/training/ortmodule/options.py

```diff
@@ -285,3 +290,22 @@ def _override_from_env_vars(self):
                 lambda x, y: x | y,
                 [_SkipCheck[name] for name in parse_os_env_skip_check_flags("ORTMODULE_SKIPCHECK_POLICY")],
             )
+
+        # Configuration for Triton.
+        # Enable Triton op executor if Triton is installed, backend has support and environment variable is set.
+        if (
+            "ORTMODULE_USE_TRITON" in os.environ
```

Comment:
Add these options and some guidance here: https://github.com/microsoft/onnxruntime/blob/main/docs/ORTModule_Training_Guidelines.md